tools/src/publish-packages/tasks/packPackageToTarball.ts

```diff
@@ -32,6 +32,9 @@ export const packPackageToTarball = new Task<TaskArgs>(
         } catch (error) {
           step.fail();
           logger.error(error.stderr);
+          if (process.env.CI) {
+            throw new Error('Failed to pack packages to tarballs');
+          }
```

Comment:
```suggestion
          throw error;
```

maybe just rethrow the error. it doesn't necessarily have different behavior between ci and running on local. let's just align the behavior.

Reply:
either way (rethrow or just throw with custom error) works, at least it stops with some error message.